### DISCUSSION TITLE
Change number of swatches on a row

### DISCUSSION THREAD

**Discussion by isohelio** [mdbassit/Coloris#166] [General]
Hi,

Is it possible to change the number of color swatches that are displayed on a row?

I took a look through the source and css and couldn't see where it might be done.

For instance, I would like to have, say, 10 swatches on a row the chooser opens up.

Thanks for your help,
Mike

**Comment by mdbassit** [comment]
The number of swatches per raw is a result of the width of the color picker's popover. Use CSS to increase the width if you want to increase the number of swatches per raw. For example:

```css
.clr-picker {
  width: 312px;
}
```

**Comment by isohelio** [comment]
Great thanks.
